### DISCUSSION TITLE
When is it coming out on 1.20?

### DISCUSSION THREAD

**Discussion by Aggelusi** [dmulloy2/ProtocolLib#2441] [Q&A]
Does anyone have any idea whether this plugin works on 1.20 and if not when are we getting an update?
Thanks

**Comment by lukalt** [comment]
Hi, you can use the latest development build for 1.20/1.20.1: https://ci.dmulloy2.net/job/ProtocolLib/654/. support. Please not that Plugins depending on ProtocolLib probably also need to be updated.
